pkg/deploy-manager/arbiter.go

```diff
@@ -0,0 +1,67 @@
+package deploymanager
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// electArbiterZone is a helper function just for the tests. In real deployments, the arbiter zone will be picked by the user.
```

Comment:
If this is just for tests, can it be included in the test code? Or at a minimum, let's include `Test` in the function name.

Reply:
This code belongs to a different package in the ocs-operator, deploy-manager which is sort of a replacement/alternative to the openshift console. Therefore is it not prefixed with `test`. It is not compiled into the ocs-operator binary though.